### PR TITLE
Arboreans

### PR DESCRIPTION
**Pull request by daedela**

### CODE REVIEW

File: src/chars.cpp [hunk 1]
```diff
@@ -88,6 +88,7 @@ int class_ok_race[NUM_RACES][NUM_CLASSES] = {
     /* Seelie Faerie         */ {Y, Y, Y, Y, N, N, Y, Y, N, N, N, N, Y, N, N, N, N, Y, Y, Y, Y, Y, Y, N},
     /* Unseelie Faerie       */ {Y, Y, Y, N, N, N, N, Y, N, N, N, Y, Y, N, N, N, N, Y, Y, Y, Y, Y, Y, N},
     /* Nymph                 */ {Y, Y, N, Y, N, N, Y, Y, Y, N, N, N, Y, N, N, N, N, Y, Y, Y, Y, Y, Y, N},
+    /* Arborean              */ {N,	Y, N,	Y, N,	N, Y,	Y, Y,	N, N,	N, N,	Y, N,	N, N,	N, N,	N, N,	N, N,	Y},
```
Comment (stridera) [inline src/chars.cpp:91]: formatting is wonky here.

File: src/races.cpp [hunk 2]
```diff
@@ -1243,7 +1243,42 @@ RaceDef races[NUM_RACES] = {
...
+     {0, 0}},                   /* function numbers - always set as {0, 0}}, to close struct */
+
+    /* ARBOREAN */
+    {"arborean",                /* name as found by triggers */
+     "arborean",                /* all race keywords */
+     "&2Arborean&0",            /* name as displayed at character creation, on who, and score */
+     "&2Arborean&0",            /* name as displayed in show race command and when setting races */
+     "Arborean",                /* name as displayed in medit vsearch stat and enlightenment */
+     false,                      /* playable? */
+     true,                      /* humanoid? */
```
Comment (stridera) [inline src/races.cpp:1255]: Are they humanoids?

**Review by stridera — APPROVED** [review]

File: src/races.cpp (hunk 2 above)
Comment (daedela) [reply src/races.cpp:1255]: yes, they're basically Ents.  They also need to be humanoid or else there are a bunch of class skills they can't use...

File: src/chars.cpp (hunk 1 above)
Comment (daedela) [reply src/chars.cpp]: How weird, it didn't do that on VSCode...

**Pull request merged**